### ISSUE TITLE
Large no. of False positives and miss detection as compared to original darknet model

### ISSUE THREAD

**Issue by muhammad-maaz-confiz** [zzh8829/yolov3-tf2#101]
Hi,

I have converted my custom (20 class) tiny yolo model from darknet weights to tensorflow files using convert.py script. The detection results are not as accurate as when I run the same model using darknet and .weights file.

Does convert.py script also performs some optimizations for inference? Why the results are not same? How can I get the same results here in TensorFlow as were using darknet?

Thanks

**Comment by AnaRhisT94** [comment]
Hi,
same here.
You will need to retrain your model with the data you trained your darknet model in order to achieve a similar accuracy again.

**Comment by muhammad-maaz-confiz** [comment]
Hi @AnaRhisT94,

I trained a yolo 20 class model using VOC2012 dataset and now I'm getting warnings that not all weights are being used while loading from checkpoints. Why is it so? How did you solve this? 

Thanks for your help

**Comment by zzh8829** [comment]
Please see training instruction here https://github.com/zzh8829/yolov3-tf2/blob/master/docs/training_voc.md
I was able to get ok quality after a few epochs
